src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,76 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
         // test your code here
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
```

Comment:
Let's move this driverService variable to the class level.
Please do the same for all service variables that generated with using injector.